Package.swift

```diff
@@ -27,6 +27,7 @@ let package = Package(
         .Package(url: "https://github.com/IBM-Swift/Kitura.git", majorVersion: 1, minor: 3),
         .Package(url: "https://github.com/IBM-Swift/HeliumLogger.git", majorVersion: 1, minor: 3),
         .Package(url: "https://github.com/IBM-Swift/Kitura-MustacheTemplateEngine.git", majorVersion: 1, minor: 1),
-        .Package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", majorVersion: 1, minor: 3),
+        .Package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", majorVersion: 1, minor: 4),
+        .Package(url: "https://github.com/kylef/Stencil", majorVersion: 0, minor: 7)
```

Comment:
@youming-lin I still do not see what is the problem. Suppose you have module A that depends on module B and module C, and module B also depends on module C. You state in Package.swift of module A which version of C you expect, and the author of the module B specifies in its Package.swift which version of C module B expects. SPM will fail if the expectations of the versions are different - you get additional check. What is wrong with that?